app/helpers/languages_helper.rb

```diff
@@ -210,8 +212,10 @@ module LanguagesHelper
   # names, but for some translations, we need the names of the
   # regional variants specifically
   REGIONAL_LOCALE_NAMES = {
+    'en-GB': 'English (British)',
     'es-AR': 'Español (Argentina)',
     'es-MX': 'Español (México)',
+    'fr-QC': 'Français (Canadien)',
```

Comment:
This is actually not the correct locale value. It should be `fr-CA`, but I'm not sure how to change that because it is what is setup in Crowdin right now. I don't think renaming the files will have the intended effect